src/styles/blocks/nav.scss

```
@@ -0,0 +1,48 @@
+.nav {
+  display: flex;
+
+  &__list {
+    display: flex;
+    margin: 0;
+    padding: 0;
+    list-style: none;
+    align-items: stretch;
+  }
+
+  &__item {
+    display: flex;
+    align-items: stretch;
+  }
+
+  &__item:not(:first-child) {
+    margin-left: 20px;
+  }
+
+  &__link {
+    display: flex;
+    align-items: center;
+    text-decoration: none;
+    color: $color-dark;
+    text-transform: uppercase;
+  }
+
+  &__link:hover {
+    color: $color-blue;
+  }
+}
+
+.is-active {
+  position: relative;
+  color: $color-blue;
+}
+
+.is-active::after {
+  content: "";
+  display: block;
+  position: absolute;
+  bottom: 0;
+  width: 100%;
+  height: 4px;
+  background-color: $color-blue;
+  border-radius: 8px;
+}
```

you can use the power of nesting
```suggestion
.is-active {
  position: relative;
  color: $color-blue;
  
  &::after {
    content: "";
    display: block;
    position: absolute;
    bottom: 0;
    width: 100%;
    height: 4px;
    background-color: $color-blue;
    border-radius: 8px;
  }
}
```